pom.xml

```diff
@@ -377,6 +377,19 @@
             </plugins>
         </pluginManagement>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
fwiw I know we don't do that already but I wonder if we should start adding comments to each plugin we add to our build with some minimal description, link to [its doc](https://maven.apache.org/plugins/maven-javadoc-plugin/jar-mojo.html) and which lifecycle phase it attaches to?  

I can imagine it's tedious that people unfamiliar with the specific plugins have to google it, get its documentation and start figuring out which lifecycle phase will this plugin attach to

Reply:
Maybe worth while for random plug-ins but for standard maven plug-ins in `org.apache.maven.plugins` its seems overkill.